content/ja/docs/reference/glossary/helm-chart.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Helm Chart
+id: helm-chart
+date: 2018-04-12
+full_link: https://helm.sh/docs/topics/charts/
+short_description: >
+  Helmツールで管理できる、事前構成されたKubernetesリソースのパッケージ。
+
+aka: 
+tags:
+- tool
+---
+ Helmツールで管理できる、事前構成されたKubernetesリソースのパッケージ。
+
+<!--more--> 
+
+チャートは、Kubernetesアプリケーションを作成および共有する再現可能な方法を提供します。
+単一のチャートを使用して、memcachedポッドなどの単純なもの、またはHTTPサーバー、データベース、キャッシュなどを含む完全なWebアプリスタックなどの複雑なものをデプロイできます。
```

Comment:
```suggestion
単一のチャートを使用して、memcached Podなどの単純なもの、またはHTTPサーバー、データベース、キャッシュなどを含む完全なWebアプリスタックなどの複雑なものをデプロイできます。
```
It is preferred to use "Pod" instead of "ポッド".